recursion/recursion_exercise.py

```diff
@@ -0,0 +1,17 @@
+# Collatz conjecture
```

Comment:
please clean up your branch state before making your branches, you should not be having noise like this in PRs especially if its work in another PR

Reply:
you can just delete it in this branch. your other PR will merge in the correct changes to put them in master later so you will not lose work.